config/opal_load_platform.m4

```diff
@@ -155,13 +172,15 @@ AC_DEFUN([OPAL_LOAD_PLATFORM], [
 
                     AC_MSG_NOTICE([Platform patches applied, created stamp file ${patch_done}])
                     touch ${patch_done}
+                else
+                    AC_MSG_NOTICE([No platform patches in ${patch_dir}])
                 fi
 
             else
                 AC_MSG_WARN([Platform patches already applied, skipping. ${patch_done} can be removed to re-apply ])
             fi
-        else
-            AC_MSG_NOTICE([No platform patches in ${patch_dir}])
+          else
```

Comment:
Super minor nit: I think the indenting of "else" was already correct; now it's incorrect.